nixos/modules/services/audio/mpd.nix

```diff
@@ -62,6 +60,14 @@ in {
         '';
       };
 
+      playlistDirectory = mkOption {
+        type = types.path;
+        default = "${cfg.dataDir}/playlists";
```

Comment:
The problem here is that if the user changes `dataDir`, the nixos manual will get rebuilt.

You should add `defaultText = ''''${dataDir}/playlists'';` to fix this.